zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt

```diff
@@ -188,7 +189,12 @@ class ZiplineLoader internal constructor(
       receive(ZiplineLoadReceiver(zipline), manifest, applicationName)
 
       // Run caller lambda to validate and initialize the loaded code to confirm it works.
-      initializer(zipline)
+      initializer(zipline, manifest)
+
+      // Run the application after initializer has been run on Zipline engine.
+      if (runAfterInitializer) {
```

Comment:
For follow-up, I wonder if we can make this always true? Or maybe, always true if `mainFunction` is not null

Reply:
I didn't know if there'd be a case where we don't want the app immediately booted so I left it configurable. If that's not a case we'd ever want, happy to simplify and do it by default.

Reply:
I think we make it on-always for now and add configurability later if we need it.